### ISSUE TITLE
Loss Weight

### ISSUE THREAD

**Issue by 980202006** [kylesargent/FlowMo#3]
Your work is very inspiring.
I noticed that the paper mentioned that the weight of the perception loss is very important. Can you share the loss curve? I want to use this loss in other tasks and want to know the value of the perception loss relative to the flow loss.

![Image](https://github.com/user-attachments/assets/ca710070-6d16-4031-93d1-4eef1e756052)

**Comment by kylesargent** [comment]
Thanks for your interest! Can you explain your question a bit more? The weights of the perceptual loss are already given in the `base.yaml` config for pretraining (.1). For post-training we use .01.

**Comment by kylesargent** [comment]
Oh I see, you need the actual loss curves.

Here is what the flow loss and perceptual loss look like for the first ~500k steps of training flowmo lo:

<img width="630" alt="Image" src="https://github.com/user-attachments/assets/327ec283-2fac-4cb9-9485-f532652b8905" />

**Comment by 980202006** [comment]
![Image](https://github.com/user-attachments/assets/fa50a99f-505f-4969-a078-d7d89cac26ac)
I use a method similar to flowmo to reconstruct audio tokens. If I set the perceptual loss to 0.01, the loss will become -0.005, and the flow loss will be around 0.9. I am worried whether this ratio is too small, so I want to know your loss ratio. The above figure is my loss, and this curve is the curve of perceptual loss and flow loss 1:1.
Thank you!

**Comment by kylesargent** [comment]
Can I close this issue? Do the plots I posted above address your question regarding the loss ratios for standard FlowMO training?

**Comment by 980202006** [comment]
Yes!Thank you!
